init/setup-rw.sh

```diff
@@ -55,15 +62,17 @@ EOF
 # file is used only if the VM has any PCI device assigned. Modules will be
 # automatically re-loaded after resume.
 EOF
+        if [ -n "$enable_selinux" ]; then restorecon -RF /rw/config; fi
```

Comment:
`qubes-relabel-rw.service` (which is now also enabled in DispVMs) will do that again. if `restorecon` is called here (and in few other cases below), maybe create `/rw/.autorelabel` too (which BTW is misnamed...).